### PR TITLE
Update chapter-02.md

### PR DESCRIPTION
**Pull request by pggallagher**
Please pardon the lack of format. I have never used Github before having trouble maneuvering my essays to format correctly.

### CODE REVIEW

File: book/chapter-02.md [hunk 1]
```diff
@@ -1,16 +1,29 @@
 # Chapter 2: Client-Side Validation, Networks & Double Spends
+What is client side validation, and why is it so important? Well, currently all blockchain consensus protocols (e.g. Bitcoin, Ethereum, Ripple, etc.) all have a challenging limitation. And that is the fact that every fully participating node in the network must process every transaction. This is client side validation. It’s the validation of inputs (blocks) to the blockchain from the nodes overseeing the network. 
 
-**What is it?**
```
Comment (eldertree) [inline book/chapter-02.md]: please disregard all red highlights

Comment (eldertree) [reply book/chapter-02.md]: My writing only goes up to the "Privacy" bullet point relating to client side validation on the outline. Please feel free to critique - I added an interesting analogy/philosophy relating to client side validation and ASICS, but perhaps this is not the place in the book to be talking about it. Might be better suited for the "future" section?

File: book/chapter-02.md [hunk 2]
```diff
@@ -1,16 +1,29 @@
...
+
+Indeed, just as a few folks bought CPUs to mine, every purchase of an ASIC has been with the intention of rigging the system for the individual. The Founding Fathers and Satoshi both imagined a world where tech would create decentralization. But because they used tech, the opposite has happened. While we cross our fingers that one day, everyone will have an equal power to mine cryptocurrency. It does not seem likely. 
+
+The Founding Fathers were philosophers, not gun toters. And like Satoshi, they were chasing a dream of decentralization. But they both seemed to make the same mistake – they relied on technology. Decentralization, however, will always rely on people.
```
Comment (pet3r-pan) [inline book/chapter-02.md]: Its ironic that you say not gun toters. Timothy C. May one of the founding members of the cypherpunks loved guns and grew up with them - hardcore libertian

Comment (pet3r-pan) [inline book/chapter-02.md]: I saw disagree. Trust relies on people and thus unreliable in itself. That is why we turn to technology where we instead put our trust in mathematics and cryptography as opposed to human trust.

**Comment by pet3r-pan** [comment]
Good start!

File: book/chapter-02.md (hunk 2 above)
Comment (pad-pad) [reply book/chapter-02.md]: Good point - couldn't agree more. I seemed to veer away from the whole basis cryptoeconomics lies on. Where traditional econonomics relies on people making rational decisions that will provide the most optimal level of benefit or utility - cryptoeconomics realizes that is a pretty big assumption and not reliable. But it was the best we had at the time. 

Where we used to rely on rational people making rational decisions every time,  Cryptoeconomics uses mathematics and cryptographic proofs as opposed to human trust. Creating a neutral system humans can actually trust. 

Will update later tonight after work, thanks Peter.

Comment (pad-pad) [reply book/chapter-02.md]: Funny you say that too, found this: 
image

Comment (pad-pad) [reply book/chapter-02.md]: image

Comment (pad-pad) [reply book/chapter-02.md]: written by Timothy May in 1994. That man was ahead of his time. 
Will think about that analogy I wrote about - might take it out. More effective ways to get the point across

Comment (pet3r-pan) [reply book/chapter-02.md]: yeah, I thought the analogy was fine ~ just wanted to comment how May was a huge gunfreak lol

**Pull request closed without merging**